tests/integration/api/test_aws_logs.py

```diff
@@ -0,0 +1,78 @@
+import os
+from datadog import api as dog
+from datadog import initialize
+
+API_KEY = os.environ.get("DD_TEST_CLIENT_API_KEY", "a" * 32)
+APP_KEY = os.environ.get("DD_TEST_CLIENT_APP_KEY", "a" * 40)
+API_HOST = os.environ.get("DATADOG_HOST")
+TEST_ACCOUNT_ID = "123456789101"
+TEST_ROLE_NAME = "DatadogApiTestRole"
+TEST_LAMBDA_ARN = "arn:aws:lambda:us-east-1:123456789101:function:APITest"
+
+
+class TestAwsLogs:
+
+    @classmethod
+    def setup_class(cls):
+        """ setup any state specific to the execution of the given class.
+        """
+        initialize(api_key=API_KEY, app_key=APP_KEY, api_host=API_HOST)
+        dog.Aws.create(
+            account_id=TEST_ACCOUNT_ID,
+            role_name=TEST_ROLE_NAME
+        )
+
+    @classmethod
+    def teardown_class(cls):
+        """ teardown any state that was previously setup with a setup_method
+        call.
+        """
+        dog.Aws.delete(account_id=TEST_ACCOUNT_ID, role_name=TEST_ROLE_NAME)
+
+    def test_list_log_services(self):
+        output = dog.AwsLogs.list_log_services()
+        assert len(output) >= 6
```

Comment:
What is that 6 ?

Reply:
That's the number of available services. This is the same for everyone and always returns this. We may add more in the future, hence the "greater than or equal to", but we're very unlikely to reduce that. I'll make this a variable instead for clarity.